internal/config/file.go

```diff
@@ -312,7 +313,7 @@ func UpsertGlobalOrganizationID(organizationID string) error {
 	return nil
 }
 
-func UpsertProjectID(projectID string) error {
+func UpsertProject(project models.Project) error {
```

Comment:
ACTUAL BUG FIX: This is where the main fix for the card is. We take the entire project instead of just the project ID, and then update the various fields. Same with `UpsertGlobalProject` a little further down.